Assets/Placenote/LibPlacenote/LibPlacenote.cs

```diff
@@ -871,6 +1075,13 @@ public PNFeaturePointUnity[] GetTrackedFeatures ()
 		return map;
 	}
 
+	/// <summary>
+	/// Public function to set the localizatino status
+	/// </summary>
+	public void SetLocalization (bool mLocalization){
```

Comment:
I don't think we should expose it as a public function, as it doesn't actually do anything for the user. The only place I see this being called is during loadmap, so why don't we set localization to true in loadmap?